packages/aws-cdk-lib/aws-sns/lib/delivery-policy.ts

```diff
@@ -0,0 +1,127 @@
+import { Duration } from '../../core';
```

Comment:
Could the contents of `delivery-policy.ts` be used elsewhere? If it’s only going to be used in `subscription.ts`, I thought it might be better to include everything in `subscriptions.ts`.